src/routes/+page.svelte

```diff
@@ -520,66 +520,54 @@
             {#each my_list as anime}
                 <a
                     href="/mal/{anime.id}/episode/{anime.current_episode}"
-                    class="relative col-span-1 flex flex-col gap-2 md:gap-[0.5vw]"
+                    class="relative"
+                    use:tippy={{
+                        arrow: false,
+                        allowHTML: true,
+                        placement: "right-start",
+                        offset: [7, 10],
```

Comment:
make the `offset` : `[0,10]`

Reply:
I added `7` coz it looks better having a space on top rather than showing on very top.
we've space below, so no issue with hovering next elements.

Reply:
> I added `7` coz it looks better having a space on top rather than showing on very top. we've space below, so no issue with hovering next elements.

I dont think adding space to the top is a very good idea

Reply:
> I dont think adding space to the top is a very good idea

No, believe me its better having a space on top.
We can see a colored thingy on top of it ( next card bg image ). which looks good

Reply:
> We can see a colored thingy on top of it ( next card bg image ). which looks good

Which is distracting lol. Thats the reason i wanted this to move to top